inference_rules.adoc

```diff
@@ -428,6 +428,79 @@ division allows calibration for quantization and does not allow any retraining.
 The unqualified name “MLPerf” must be used when referring to a Closed Division
 suite result, e.g. “a MLPerf result of 4.5.”
 
+=== Network Division
+
+The Network division inherits all requirements from the Closed division and imposes further constraints.
+In the Network division the SUT is connected to the Loadgen system over a network fabric. The Query Dispatch Library (QDL) component is a submitter-implemented SUT proxy that runs on the Loadgen system. The Network division supports only the Datacenter suite.
+
+==== QDL Constraints
+
+* The QDL is not allowed to do any pre-processing.
+* If an SUT compresses its output, the QDL must decompress the output. Decompression is a timed operation. No other post-processing in the QDL is allowed.
+* The QDL is not allowed to batch queries.
+* The Name method's return value must contain the substring "Network SUT".
+* The Name method's implementation must include at least one round trip over the network. The Name method must not return until the round trip is complete.
+
+==== General Constraints
+
+MLPerf distinguishes between fabric interconnects and bus interconnects. Fabrics interconnects are required and bus interconnects are forbidden.
+
+A fabric interconnect must:
+
+1. Work as out-of-the-box chassis-to-chassis interconnect
+1. Use wireless, copper, or fiber-optics media
+1. Suitable for connecting systems above 10 meters distance
+1. Use switch topology
+1. Be highly scalable, reliable, and fault-tolerant
```

Comment:
Does the fabric interconnect technology also need to be "available" for purchase/use on-prem or in the cloud?

Reply:
Only in the Available division, right?

Reply:
As discussed in the task force this text is not needed.